core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -33,6 +33,9 @@ public class TableProperties {
 
   private TableProperties() {
   }
+  
+  public static final String TABLE_VERSION = "table.version";
+  public static final String TABLE_VERSION_DEFAULT = "0.5.0";
```

Comment:
Has this configuration been used before? Using the project version number as the table version doesn't seem like a good choice.

Reply:
It's useless code, and I will remove that.